src/lib/config/tokens-list/index.ts

```diff
@@ -0,0 +1,16 @@
+import { TokenListURLMap } from '@/types/TokenList';
+
+const tokenlists: TokenListURLMap = {
+  Balancer: {
+    Default:
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/balancer.tokenlist.json',
+    Vetted:
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json',
+  },
+  External: [
+    'ipns://tokens.uniswap.org',
+    'https://www.gemini.com/uniswap/manifest.json',
+  ],
+};
```

Comment:
I get why you've done this but please don't change the way the config works in this PR, just replace the existing URLs with these ones in all the different network files for tokenlists.